pkg/manager/client/discovery.go

```diff
@@ -3,9 +3,9 @@ package client
 import (
 	"context"
 	"fmt"
+	"io"
```

Comment:
Weird. This should be automatically corrected by `gofmt`. @shangma Have you got `gofmt` setup in your IDE?

Reply:
@yasker  Just did. Many thanks for the reminder.